spec/motion-support/core_ext/time/conversion_spec.rb

```diff
@@ -1,50 +1,49 @@
-describe "Time" do
```

Comment:
Totally fine, force of habit from work ("if you see it, fix it"), but also definitely didn't know it was the convention of this project. Would recommend that we set up a Rubocop and tie it into Hound so that people don't have to think about that sort of thing, they're automatically notified.

They should all be double quotes now.